src/ocm/components/clusterConfiguration/OcmClusterDetailsFormFields.tsx

```diff
@@ -82,6 +82,7 @@ export const OcmClusterDetailsFormFields = ({
             ? uniqueOcmClusterNameValidationMessages(t)
             : ocmClusterNameValidationMessages(t)
         }
+        maxLength={54}
```

Comment:
It would be better to introduce a new constant for the maxLength we allow and to use it here, and also in `nameValidationSchema`

If it ever changes, we won't need to remember to keep the two in sync or to find all references which need to be updated.